astropy/utils/introspection.py

```diff
@@ -143,22 +139,19 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     else:
         have_version = resolve_name(module.__name__, version_path)
 
-    try:
-        from pkg_resources import parse_version
-    except ImportError:
-        from distutils.version import LooseVersion as parse_version
-        # LooseVersion raises a TypeError when strings like dev, rc1 are part
-        # of the version number. Match the dotted numbers only. Regex taken
-        # from PEP440, https://www.python.org/dev/peps/pep-0440/, Appendix B
-        expr = '^([1-9]\\d*!)?(0|[1-9]\\d*)(\\.(0|[1-9]\\d*))*'
-        m = re.match(expr, version)
-        if m:
-            version = m.group(0)
+    from distutils.version import LooseVersion
+    # LooseVersion raises a TypeError when strings like dev, rc1 are part
```

Comment:
I don't know if this is true anymore:

```python
In [5]: LooseVersion('3.0dev0') < LooseVersion('3.0dev1')
Out[5]: True
```

Reply:
(this was added in https://github.com/astropy/astropy/pull/5944 so we should check why)

Reply:
Good point. All tests pass on python 3.7 at least without it. I'll push a new commit that removes the doctoring - we'll see if the tests #5944 added pass on all versions.